website-custom-views/src/content/api-reference/commercetools-frontend-cypress.mdx

```diff
@@ -73,37 +73,24 @@ import '@commercetools-frontend/cypress/add-commands';
 
 # Commands
 
-## cy.loginToMerchantCenter
+## cy.loginToMerchantCenterForCustomView
 
-This command performs the user login to the Merchant Center. It automatically detects whether the Custom View is running on localhost or production and chooses the appropriate login mechanism.<br/>
-
-- When it runs locally, the same mechanism used in the `cy.loginByOidc` is used.
-- When it runs on production, a normal login flow is used where the user enters the credentials into the login form.
+This command performs the user login to the Merchant Center, designed specifically for testing a Custom View. Testing Custom Views is expected to occur on localhost, therefore the `cy.loginByOidc` mechanism is used.<br/>
 
 ### Usage
 
 ```ts
-import { CUSTOM_VIEW_HOST_ENTRY_POINT_URI_PATH } from '@commercetools-frontend/constants';
-
 it('should render page 1', () => {
-  cy.loginToMerchantCenter({
-    entryPointUriPath: CUSTOM_VIEW_HOST_ENTRY_POINT_URI_PATH,
-    initialRoute: `/${Cypress.env('PROJECT_KEY')}/my-app/page-1`,
-  });
-})
-it('should render page 2', () => {
-  cy.loginToMerchantCenter({ entryPointUriPath: CUSTOM_VIEW_HOST_ENTRY_POINT_URI_PATH });
-  cy.visit(`/${Cypress.env('PROJECT_KEY')}/my-app/page-2`)
+  cy.loginToMerchantCenterForCustomView();
 })
 ```
 
 ### Options
 
 Available options are:
 
-* `entryPointUriPath` (**required**): User the constant for the entry point URI path: `CUSTOM_VIEW_HOST_ENTRY_POINT_URI_PATH`.
-* `dotfiles` (optional): A list of dotenv files to load when the `custom-view-config.mjs` is loaded (in case you're using an environment placeholder). By default the following dotenv files are loaded: `.env` and `.env.local`. You can also define the values using paths relative to the application folder.
-* `initialRoute` (optional): The route to open after login. If not defined, make sure to call `cy.visit` yourself.
+* `packageName` (optional): The package name as specified in the Custom View's `package.json`. If not provided, it defaults to `Cypress.env('PACKAGE_NAME').`
+* `dotfiles` (optional): A list of dotenv files to load when the `custom-view-config.mjs` is loaded (in case you're using an environment placeholder). By default the following dotenv files are loaded: `.env` and `.env.local`. You can also define the values using paths relative to the Custom View folder.
```

Comment:
@kark what do you mean by "environment placeholder" here? Can you give an example, currently it sounds ambiguous.

Reply:
Hi I found what you mean, I added the link (/api-reference/custom-view-config#using-variable-placeholders). let me know if that's correct.